dist/index.js

```diff
@@ -12757,6 +12757,11 @@ function getInputs() {
         if (isWorkflowRepository) {
             result.ref = github.context.ref;
             result.commit = github.context.sha;
+            // Some events have an unqualifed ref. For example when a PR is merged (pull_request closed event),
```

Comment:
@andymckay is this a bug on the server? Seems like github.ref would always be qualified (e.g. `refs/heads/master` not `master`). For most events it's qualified. PR closed_event (when a PR is merged) is the only case I'm aware it's not.

I'm still planning to proceed with this fix here for resiliency, but just FYI since this seems like a bug on the server.